be2-scala/linter/rules/src/main/scala/fix/Util.scala

```diff
@@ -31,18 +53,38 @@ object Util {
     }.headOption.orNull
   }
 
+  /** Find multiple definitions in one tree traversal.
+   * @param tree The tree to search in
+   * @param nameSet The set of names to search for
+   * @return An unordered list of definitions found
+   */
+  // Takes a set as argument as we will have to do many lookups
   def findDefinitions(tree: Tree, nameSet: Set[Term]): List[(Term, Any)] = {
     tree.collect {
       case Defn.Val(_, List(Pat.Var(varName)), _, value) if nameSet.exists(_.toString().equals(varName.value)) => nameSet.find(_.toString().equals(varName.value)).get -> value
       case Defn.Var.After_4_7_2(_, List(Pat.Var(varName)), _, value) if nameSet.exists(_.toString().equals(varName.value)) => nameSet.find(_.toString().equals(varName.value)).get -> value
     }
   }
 
-  // Finds multiple definitions in one tree traversal
+  /**
+   * Finds multiple definitions in one tree traversal, returning a map of the definitions
+   * @param tree The tree to search in
+   * @param nameSet The set of names to search for
+   * @return A map of the definitions
+   */
   def findDefinitionsMap(tree: Tree, nameSet: Set[Term]): Map[Term, Any] = {
     findDefinitions(tree, nameSet).toMap
   }
```

Comment:
Should have seen it sooner but, do we need both of them ? especially since the the latter is just a `.toMap`

Reply:
I believe it should be fine given the one-liner it is today, I approved a bit fast so you are free to keep it or remove it.

Reply:
Removed it :)